internal/cli/main.go

```diff
@@ -29,6 +29,10 @@ func Run(args []string) bool {
 		migrateCmd     = app.Command("migrate", "migrate command")
 		migrateUpCmd   = migrateCmd.Command("up", "migrate db up")
 		migrateDownCmd = migrateCmd.Command("down", "migrate db down")
+
+		event    = app.Command("event", "claim event command")
+		eventCmd = event.Command("start", "claim event command")
+		data     = eventCmd.Arg("data", "data after ...").Required().Int()
```

Comment:
Change command names to more understandable: `event create-early-test -before 1730000`